README.md

```diff
@@ -1 +1,190 @@
-# inpycon2021-tasks
\ No newline at end of file
+# PyCon India 2021
+
+
+## Event Dates:
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
+    * Diversity
+    * Extended
+- Multiple Sessions
+- Community Booths
+- Bofs
+
+## Working Groups
+
+Conference Chair: Chandan Kumar
+Conference Co-Chairs: Kumar Anirudha, Ritesh Agarwal
+Emritus Chair: Sayan Chowdhury
```

Comment:
s/Emritus/Emeritus

Also, add a comment that previous chairs would be reached out (if we planning to add more, it's okay to limit to one or two). Also, we need to create a doc on the roles. This is for all the tasks roles.